packages/fxa-settings/src/components/CardHeader/index.test.tsx

```diff
@@ -36,7 +36,7 @@ describe('CardHeader', () => {
 
     expect(
       screen.getByRole('heading', {
-        name: `${MOCK_HEADING} to continue to ${MozServices.Default}`,
```

Comment:
Sorry if this is a silly question -- why are we hard-coding this?

Reply:
I'm inclined to test against what I expect to see as output vs. a variable that may have changed and that I don't control in the test. E.g., if the variable was changed outside this source code, it might no longer be what we want to display on this specific page.

Also, I felt it would be good to check that there is no change to the expected `MozServices.Default` since "to continue to account settings" is localized. Once we get the l10n test suite up and running though this check shouldn't be needed.

Reply:
Ohhh interesting! Sounds good to me!